src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -612,12 +653,101 @@ private void WriteObjectsOrShowOnlineHelp(HelpInfo helpInfo, bool showFullHelp)
                             }
                         }
 
-                        WriteObject(helpInfo.ShortHelp);
+                        WriteObjectOrWritePaged(helpInfo.ShortHelp, bufferOutputWhenPaged: true);
                     }
                 }
             }
         }
 
+        private void WriteObjectOrWritePaged(object sendToPipeline, bool bufferOutputWhenPaged)
+        {
+            if (Paged)
+            {
+                if (pager is null)
+                {
+                    pager = new Pager();
+                }
+
+                var helpText = GetHelpOutput();
+
+                if (bufferOutputWhenPaged)
+                {
+                    multiItemPagerBuffer = helpText;
```

Comment:
Should we accumulate?
```suggestion
                    multiItemPagerBuffer += helpText;
```

Reply:
Added comment to explain why appending is not needed.